src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
```

Comment:
**All instances of the Animal class must have properties health and a name. Health is 100 by default.**
I think `health` should be 100 only by default, if another value is passed into the constructor it should be different.

Also, let's avoid magic values such as 100, 50